_includes/feature-phases/azure-limited-access.md

```diff
@@ -0,0 +1,6 @@
+{% comment %}This is a fork of limited-access.md specific to Azure on CC Dedicated{% endcomment %}
+**Creating a {{ site.data.products.dedicated }} cluster on Microsoft Azure is in [limited access](/docs/{{site.versions["stable"]}}/cockroachdb-feature-availability.html)** and is only available to enrolled organizations. To enroll your organization, contact your Cockroach Labs account team.
+
+The [{{ site.data.products.db }} SLA](https://cockroachlabs.com/cloud-terms-and-conditions/cockroachcloud-technical-service-level-agreement/) does not apply to clusters on Microsoft Azure. Technical support is limited to standard business hours. For more information and limitations, refer to [{{ site.data.products.dedicated }} on Azure](https://cockroachlabs.com/docs/cockroachcloud/cockroachdb-dedicated-on-azure.html#faq).
```

Comment:
Yeah I added the #faq anchor because I'd previously given it to you. I'll change it here and if you are linking to the FAQ anywhere can you update your links too?